decko/spec/cypress/integration/edit/pointer.spec.coffee

```diff
@@ -56,7 +56,7 @@ describe 'editing pointers', () ->
         .should("contain", "Joe Camel")
       cy.contains(/select\s+3\s+following/)
       cy.get("input._select-all").click()
-      cy.contains(/select\s+0\s+following/)
+      # cy.contains(/select\s+0\s+following/)
```

Comment:
this appears very briefly and then disappears, so it fails sporadically if the timing is wrong.

Reply:
Why does it appear briefly? Shouldn't it remain 0 after I selected all?

Reply:
see the `count > 0` stuff in select_item.haml for more context.

I _think_ the rationale is that if there are no results, there is no need for a checkbox.  Another way to go would be to leave the checkbox there the whole time but disable if if there are 0 items to select...